<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Быстрое пиковое обнаружение</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>gui/icons/activity.png</normaloff>gui/icons/activity.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_8" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_4" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Исходные данные</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="btn_dataview_input">
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>gui/icons/search.png</normaloff>gui/icons/search.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="btn_import">
                 <property name="toolTip">
                  <string>Просмотреть выбранный файл</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>gui/icons/database-download.png</normaloff>gui/icons/database-download.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="list_files"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget_5" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="widget_6" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Преобразованные данные</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QToolButton" name="btn_dataview_output">
              <property name="toolTip">
               <string>Просмотр данных</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>gui/icons/search.png</normaloff>gui/icons/search.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_graph">
              <property name="toolTip">
               <string>Просмотреть выбранный файл</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>gui/icons/statistics-chart.png</normaloff>gui/icons/statistics-chart.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btn_export">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>gui/icons/database-upload.png</normaloff>gui/icons/database-upload.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QListWidget" name="output_files"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_9" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Предпросмотр</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="graph_widget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>220</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Проект</string>
    </property>
    <addaction name="open_master_import"/>
    <addaction name="open_export_master"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Преобразование</string>
    </property>
    <addaction name="open_convertion_master"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="action_reader_chart"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_master_import"/>
   <addaction name="open_export_master"/>
   <addaction name="separator"/>
   <addaction name="action_reader_chart"/>
   <addaction name="separator"/>
   <addaction name="open_convertion_master"/>
  </widget>
  <action name="open_master_import">
   <property name="icon">
    <iconset>
     <normaloff>gui/icons/database-download.png</normaloff>gui/icons/database-download.png</iconset>
   </property>
   <property name="text">
    <string>Мастер импорта</string>
   </property>
   <property name="toolTip">
    <string>Мастер импорта данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="open_convertion_master">
   <property name="icon">
    <iconset>
     <normaloff>gui/icons/MD-play.png</normaloff>gui/icons/MD-play.png</iconset>
   </property>
   <property name="text">
    <string>Мастер преобразования</string>
   </property>
   <property name="toolTip">
    <string>Открыть мастер преобразования данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="open_export_master">
   <property name="icon">
    <iconset>
     <normaloff>gui/icons/database-upload.png</normaloff>gui/icons/database-upload.png</iconset>
   </property>
   <property name="text">
    <string>Мастер экспорта данных</string>
   </property>
   <property name="toolTip">
    <string>Открыть мастер экспорта данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_reader_chart">
   <property name="icon">
    <iconset>
     <normaloff>gui/icons/statistics-chart.png</normaloff>gui/icons/statistics-chart.png</iconset>
   </property>
   <property name="text">
    <string>Мастер визуализации</string>
   </property>
   <property name="toolTip">
    <string>Открыть мастер визуализации данных</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Закрыть программу</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>open_export_master</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_export_master()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_master_import</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_import_master()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_files</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>select_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_convertion_master</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_conversion_master()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_export</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_export_master()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>651</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_graph</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_visualization_master()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_import</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_import_master()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>list_files</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>select_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_dataview_output</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_dataview_output_master()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>606</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_dataview_input</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_dataview_input_master()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>283</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_reader_chart</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_visualization_master()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>open_import_master()</slot>
  <slot>open_export_master()</slot>
  <slot>open_visualization_master()</slot>
  <slot>select_file()</slot>
  <slot>open_conversion_master()</slot>
  <slot>open_dataview_output_master()</slot>
  <slot>open_dataview_input_master()</slot>
 </slots>
</ui>
